fetch.bs

```diff
@@ -7447,45 +7456,58 @@ constructor steps are:
  <li><p>If <var>init</var>["{{ResponseInit/statusText}}"] does not match the
  <a spec=http>reason-phrase</a> token production, then <a>throw</a> a {{TypeError}}.
 
- <li><p>Set <a>this</a>'s <a for=Response>response</a> to a new <a for=/>response</a>.
-
- <li><p>Set <a>this</a>'s <a for=Response>headers</a> to a <a for=/>new</a> {{Headers}} object with
- <a>this</a>'s <a>relevant Realm</a>, whose <a for=Headers>header list</a> is <a>this</a>'s
- <a for=Response>response</a>'s <a for=response>header list</a> and <a for=Headers>guard</a> is
- "<code>response</code>".
-
- <li><p>Set <a>this</a>'s <a for=Response>response</a>'s <a for=response>status</a> to
+ <li><p>Set <var>response</var>'s <a for=Response>response</a>'s <a for=response>status</a> to
  <var>init</var>["{{ResponseInit/status}}"].
 
- <li><p>Set <a>this</a>'s <a for=Response>response</a>'s <a for=response>status message</a> to
+ <li><p>Set <var>response</var>'s <a for=Response>response</a>'s <a for=response>status message</a> to
  <var>init</var>["{{ResponseInit/statusText}}"].
 
  <li><p>If <var>init</var>["{{ResponseInit/headers}}"] <a for=map>exists</a>, then
- <a for=Headers>fill</a> <a>this</a>'s <a for=Response>headers</a> with
+ <a for=Headers>fill</a> <var>response</var>'s <a for=Response>headers</a> with
  <var>init</var>["{{ResponseInit/headers}}"].
 
  <li>
-  <p>If <var>body</var> is non-null, then:
+  <p>If <var>body</var> was given, then:
 
   <ol>
    <li>
-    <p>If <var>init</var>["{{ResponseInit/status}}"] is a <a>null body status</a>, then <a>throw</a>
-    a {{TypeError}}.
+    <p>If <var>response</var>'s <a for=response>status</a> is a <a>null body status</a>, then
+    <a>throw</a> a {{TypeError}}.
 
     <p class="note no-backref">101 is included in <a>null body status</a> due to its use elsewhere.
     It does not affect this step.
 
-   <li><p>Let <var>Content-Type</var> be null.
-
-   <li><p>Set <a>this</a>'s <a for=Response>response</a>'s <a for=response>body</a> and
-   <var>Content-Type</var> to the result of <a for=BodyInit>extracting</a> <var>body</var>.
+   <li><p>Set <var>response</var>'s <a for=response>body</a> to <var>body</var>'s
+   <a for="body with type">body</a>.
 
-   <li><p>If <var>Content-Type</var> is non-null and <a>this</a>'s <a for=Response>response</a>'s
-   <a for=response>header list</a> <a for="header list">does not contain</a>
+   <li><p>If <var>body</var>'s <a for="body with type">type</a> is non-null and
+   <var>response</var>'s <a for=response>header list</a> <a for="header list">does not contain</a>
    `<code>Content-Type</code>`, then <a for="header list">append</a> (`<code>Content-Type</code>`,
-   <var>Content-Type</var>) to <a>this</a>'s <a for=Response>response</a>'s
+   <var>body</var>'s <a for="body with type">type</a>) to <var>response</var>'s
    <a for=response>header list</a>.
-  </ol>
+</ol>
```

Comment:
I think there's a missing closing tag here as we're going from 3 spaces to 0 too quickly.